models/syndicate_mission.py

```diff
@@ -1,4 +1,14 @@
-from models.common import WarframeObj
+from collections import defaultdict
+from dateutil.parser import isoparse
+from models.common import WarframeObj, Job
 
 class SyndicateMission(WarframeObj):
-    pass
\ No newline at end of file
+    def __init__(self, json_syndicate_mission: dict):
+        json_syndicate = defaultdict(lambda : None, json_syndicate_mission)
```

Comment:
You created a new variable "json_syndicate" instead of redefining "json_syndicate_mission"